explainer.md

```diff
@@ -46,207 +47,84 @@ This effort aims to create a JavaScript API
+    - In the case of something like 
+    [`aria-activedescendant`](https://www.w3.org/TR/wai-aria-1.1/#aria-activedescendant),
+    this may be one of hundreds or thousands of elements, 
+    depending on the UI.
+    This requirement makes these APIs cumbersome to use
+    and lead to many extra DOM attributes being necessary.
+3. Listening for events from Assistive Technology
+   - Currently, _only_ built-in elements have the capability to react to events,
+     typically triggered by user actions such as 
+     ["click"](https://developer.android.com/reference/android/view/accessibility/AccessibilityEvent.html#TYPE_VIEW_CLICKED) 
+     or ["increment"](https://developer.apple.com/documentation/objectivec/nsobject/1615076-accessibilityincrement).
+4. Adding non-DOM nodes (“virtual nodes”) to the Accessibility tree 
+   - For example, to express a complex UI built out of a `<canvas>` element,
+     or streaming a remote desktop to a `<video>` element, etc.
+   - These should be able to express at least the same set of accessible properties as Elements, 
+     as well as parent/child/other relationships with other virtual nodes, 
+     and position/dimensions.
+5.  Introspecting the computed tree
+   - Developers currently have no way to probe or test how ARIA and other accessible properties are applied.
 
 ## The Accessibility Object Model
 
-This spec proposes the *Accessibility Object Model* (AOM).
-We plan to split this work into four phases,
-which will respectively allow authors to:
-
-1. modify the semantic properties of the accessibility node associated with a particular DOM node,
-3. directly respond to events (or actions) from assistive technology,
-3. create virtual accessibility nodes which are not directly associated with a DOM node, and
-4. programmatically explore the accessibility tree
-   and access the computed properties of accessibility nodes.
-
-In the following sections we'll outline each of these phases.
-
-### Phase 1: Modifying Accessible Properties
-
-This phase will explain and augment the existing capabilities of
-[ARIA](https://www.w3.org/TR/wai-aria-1.1/).
-
-Using ARIA, it's possible to override and add to the default semantic values for an element:
-
-```html
-<div role="checkbox" aria-checked="true">Receive promotional offers</div>
-```
-
-In this example,
-the [default role of a `<div>`](https://www.w3.org/TR/html-aam-1.0/#el-div),
-which would usually be assigned a grouping role or excluded from the accessibility tree,
-is overridden to give the accessibility node associated with this element
-a role of `checkbox`.
-
-Furthermore, the
-[default value of the accessible `checked` property](https://www.w3.org/TR/wai-aria/states_and_properties#aria-checked)
-is overridden to expose the element as a "checked checkbox".
-
-**The first phase of AOM, *Accessible Properties*,
-would create a different mechanism to achieve the same result:**
+Initially, our intention was to combine these use cases into a read/write API
```

Comment:
Do you want to lead off with discussing the things we chose not to do? That kind of obscures things for anyone who is trying to quickly get up to speed. I might suggest instead moving this to a later section called something like "Alternative Implementations" that explains the paths we chose not to go down and why.

Reply:
Hm... my feeling was that most people reading at this stage would have some existing notion of what the API "is", and I wanted to address that directly, but perhaps I can move it a bit later.